### ISSUE TITLE
SameSite cookie

### ISSUE THREAD

**Issue by seanho00** [seanho00/reveal-multiplex#3]
Travis build using decktape to print:

```
A cookie associated with a cross-site resource at http://mp.seanho.com/ was set without the `SameSite` attribute.
A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`.
You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
```

**Comment by highlando** [comment]
does this break things? I couldn't get multiplex to work. Neither with your workflow, nor with the workflow described on the [reveal.js repo](https://github.com/hakimel/reveal.js/blob/master/README.md#multiplexing)

**Comment by seanho00** [comment]
Nope, it's just a warning for now, and even if Chrome were to reject the cookie, the multiplex server doesn't depend on cookies. It's only a very basic pub/sub with socket.io.  I have a sample site in sermons/reveal-skel that works with multiplex; you can even pull up the master in another window and control clients.

**Comment by highlando** [comment]
Yes, I saw your demo. Very nice! It convinced me to try your work flow after the original didn't work for me. I will make another try tonight. Maybe the problem was that my github pages are redirected to a custom domain. Must say that I am far from being an expert on web design. (also I can delete all my comments later. Don't want to spam your issue tracker.) 

Am 26. März 2020 17:55:18 MEZ schrieb Sean Ho <notifications@github.com>:

**Comment by highlando** [comment]
Sean -- thank you so much. I got everything running with your workflow. (Really, I'm going to remove my comments soon :) )

**Comment by seanho00** [comment]
Glad you got it worked out!  Multiplex is pretty slick, eh?  Hakim did an excellent job with reveal.js.

No worries about removing your comments; they're only slightly off-topic for this issue, and mostly I'm just surprised anyone has noticed my repos!

**Comment by highlando** [comment]
Yes, multiplex is amazing.

Actually, your repo ranked third for me, when searching for *reveal js multiplex*
